src/tools/adig.c

```diff
@@ -43,9 +43,16 @@
 #endif
 
 #include "ares.h"
+#include "ares__array.h"
+#include "ares__buf.h"
 #include "ares_dns.h"
+#include "ares_private.h"
```

Comment:
don't include ares_private.h

Reply:
How do I get "ares_free" otherwise without it?

Reply:
include ares_mem.h